src/test/scala/edu/ie3/simona/test/common/input/NodeInputTestData.scala

```diff
@@ -6,15 +6,17 @@
 
 package edu.ie3.simona.test.common.input
 
-import java.util.UUID
-
 import edu.ie3.datamodel.models.input.{NodeInput, OperatorInput}
 import edu.ie3.datamodel.models.voltagelevels.GermanVoltageLevelUtils
 import edu.ie3.simona.test.common.DefaultTestData
 import edu.ie3.util.quantities.PowerSystemUnits.PU
+import org.locationtech.jts.geom.{Coordinate, GeometryFactory, Point}
 import tech.units.indriya.quantity.Quantities
 
-/** //ToDo: Class Description
+import java.util.UUID
+
+/** Default values for NodeInput's to be used in tests. Should be extended as
+  * needed.
   *
   * @version 0.1
   * @since 23.06.20
```

Comment:
Please remove this. I don't think we really use this - or we should update the version and date...